.test-infra/jenkins/PrecommitJobBuilder.groovy

```diff
@@ -76,7 +76,7 @@ class PrecommitJobBuilder {
     }
     job.with {
       description buildDescription('for each commit push.')
-      //concurrentBuild()
```

Comment:
This change was unexpected merged to master by another PR 10 mins ago..Now I am recovering it.